manifests/image-references

```diff
@@ -5,7 +5,7 @@ spec:
   - name: cluster-kube-apiserver-operator
     from:
       kind: DockerImage
-      name: docker.io/openshift/origin-cluster-kube-apiserver-operator:v4.0
+      name: quay.io/openshift/origin-cluster-kube-apiserver-operator:v4.0
```

Comment:
@deads2k this change shouldn't break anything? service-ca-operator, cluster-authentication-operator reference quay.io